src/browser/Navigators/Navigator/WebView/MozBrowserFrame.js

```diff
@@ -185,7 +185,7 @@ const decodeLocationChange =
     , canGoForward: null
     }
   : { type: "LocationChanged"
-    , uri: detail.uri
+    , url: detail.url
```

Comment:
Oh good catch! So, no, you don't need, because the other is for Gecko, but then, the ` || detail` that you added is not necessary.

So basically, you just need to `uri: detail.uri` to `uri: detail.url || detail.uri`.